### PR TITLE
Create pi40.json

### PR DESCRIPTION
**Pull request by ziptyze**
First version of the configuration .json for 1upkeyboards pi40

## Description

This is the first version of the .json for the pi40, a keyboard by 1upkeyboards using a Raspberry Pi Pico for the controller. It uses a rp2040 microcontroller, and support has recently been added to QMK. The board is a 4x12 ortho board with per-key RGB, a rotary encoder, and an oled.

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/18207

## Checklist

- [x] The VIA support for this keyboard is in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by ziptyze** [comment]
QMK merge has occurred

File: v3/1upkeyboards/pi40/pi40.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
+{
+    "name": "pi40",
+    "vendorId": "0x6F75",
+    "productId": "0x5600",
+    "keycodes": ["via/keycodes", "via/qmk_lighting"],
+    "menus": ["qmk_rgb_matrix"],
```
Comment (Xelus22) [inline v3/1upkeyboards/pi40/pi40.json:6]: Please remove these for V2 definitions. It is not currently supported yet until V3 is merged in. https://github.com/qmk/qmk_firmware/pull/18222

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/1upkeyboards/pi40/pi40.json (hunk 1 above)
Comment (ziptyze) [reply v3/1upkeyboards/pi40/pi40.json]: I moved the folder and file the src directory, and removed those lines of code you requested.

**Comment by Xelus22** [comment]
LGTM. Thanks!

**Pull request merged**